incubator/hnc/internal/reconcilers/hnc_config.go

```diff
@@ -132,14 +132,12 @@ func (r *ConfigReconciler) reconcileTypes(inst *api.HNCConfiguration) error {
 	r.activeGR = gvk2gr{}
 	for _, t := range inst.Spec.Types {
 		gr := schema.GroupResource{Group: t.Group, Resource: t.Resource}
-		grm := fmt.Sprintf("Group: %s, Resource: %s, Mode: %s", t.Group, t.Resource, t.Mode)
 		// If there are multiple configurations of the same type, we will follow the
 		// first configuration and ignore the rest.
-		if _, exist := r.activeGVKMode[gr]; exist {
-			msg := fmt.Sprintf("Ignore the configuration: %q because the configuration of the type already exists; "+
-				"only the first configuration will be applied", grm)
+		if gvkMode, exist := r.activeGVKMode[gr]; exist {
+			msg := fmt.Sprintf("%s has multiple settings; all but one (mode: %s) will be ignored.", gr, gvkMode.mode)
```

Comment:
s/settings/synchronization modes/ ? Then you can get rid of the "mode:" in the brackets.

Reply:
Just confirming that we don't need to specify the mode is being used or ignored?

Some integration tests used to verify what modes are ignored and I changed it to verify what modes are used since we changed the msg. If removing the mode in the msg, we just verify a type has the condition. SGTY?

Reply:
Sorry, that wasn't very clear!